bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/DevModeOperations.java

```diff
@@ -349,7 +351,7 @@ public void startInContainer(IProject iProject, String parms, String javaHomePat
 
             // If there is a debugPort, start the job to attach the debugger to the Liberty server JVM.
             if (debugPort != null) {
-                debugModeHandler.startDebugAttacher(project, debugPort);
+                debugModeHandler.startDebugAttacher(project, launch, debugPort);
```

Comment:
Should we move startDevMode() in line 358 up before startDebugAttacher like we did in (non-container) start?